hyperon_das/das.py

```diff
@@ -449,7 +459,7 @@ def count_atoms(self, parameters: Optional[Dict[str, Any]] = None) -> Dict[str,
     def query(
         self,
         query: Query,
-        parameters: Optional[Dict[str, Any]] = {},
+        parameters: Optional[Dict[str, Any]] = None,
```

Comment:
It follows the same reasoning explained [here](https://github.com/singnet/das-query-engine/pull/313/files#r1716889210).
You probably mixed some code because this function has no `if` statement.